src/logging.c

```diff
@@ -1407,13 +1414,24 @@ rcutils_ret_t rcutils_logging_format_message(
 # define SET_STANDARD_COLOR_IN_STREAM(is_colorized, status)
 #endif
 
+static bool rcutils_logging_is_colorized()
+{
+  if (g_colorized_output == RCUTILS_COLORIZED_OUTPUT_FORCE_ENABLE) {
+    return true;
+  }
+  if (g_colorized_output == RCUTILS_COLORIZED_OUTPUT_FORCE_DISABLE) {
+    return false;
+  }
+  return IS_STREAM_A_TTY(g_output_stream);
```

Comment:
Fix for `rmw_zenoh_cpp` test in `test_rmw_implementation`

```suggestion
  }
  if (g_output_stream == NULL) {
    return false;
  }
  return IS_STREAM_A_TTY(g_output_stream);
```